### PR TITLE
Updates a couple of hover states and I have added additional state to…

### PR DESCRIPTION
**Pull request by Ddungyov**
… the Action item -> Selected-hover

### CODE REVIEW

**Comment by StefanIvanov** [comment]
I think we should first merge #81 because there will be some things conflicting with some of the changes on this branch that are better to be resolved within this one.

File: data/tokens.json [hunk 1]
```diff
@@ -1693,12 +1695,6 @@
       "gtmds_button_background-color-disabled": {
         "value": "rgba({White.gtmds_button_background-color},0.4)",
         "type": "color"
-      },
-      "Primary": {
-        "gtmds_button_background-color-hover": {
-          "value": "rgba({White.gtmds_button_background-color}, 0.7)",
-          "type": "color"
-        }
```
Comment (StefanIvanov) [inline data/tokens.json:1701]: Why do you suggest to remove this? We would still need a style defined for hover on a white, primary button ...

File: data/tokens.json [hunk 2]
```diff
@@ -1296,8 +1300,6 @@
       "value": "{Text.gtmds_color_text-color}",
       "type": "color"
     },
-    "gtmds_accordion_color-disabled": {
-      "value": "{Text.gtmds_color_text-color-disabled}",
```
Comment (StefanIvanov) [inline data/tokens.json:1300]: I am OK with removing this style but does that mean we have no such state for the accordion or we indicate disabled by another means e.g. hiding the expand button or something else?

Comment (Ddungyov) [reply data/tokens.json:1300]: Yes, I am not aware of having a disabled state for the accordion and such a need so far, this is why I've removed it.

File: data/tokens.json (hunk 1 above)
Comment (Ddungyov) [reply data/tokens.json:1701]: I don't remember why I removed this, probably by mistake. Let's revert this change.

File: data/tokens.json [hunk 3]
```diff
@@ -1693,12 +1695,6 @@
       "gtmds_button_background-color-disabled": {
         "value": "rgba({White.gtmds_button_background-color},0.4)",
         "type": "color"
```
Comment (StefanIvanov) [inline data/tokens.json:1697]:
```suggestion
        "type": "color"
      },
      "Primary": {
        "gtmds_button_background-color-hover": {
          "value": "rgba({White.gtmds_button_background-color}, 0.7)",
          "type": "color"
        }
```

Comment (StefanIvanov) [reply data/tokens.json:1697]: not sure if upon accepting this it would work, please verify before proceeding but if all is good you can merge :)

**Review by StefanIvanov — APPROVED** [review]

File: data/tokens.json (hunk 3 above)
Comment (StefanIvanov) [reply data/tokens.json:1697]: we risk breaking the json format but at least we will know where it came from and it should be a relatively easy fix :)

Comment (Ddungyov) [reply data/tokens.json]: There's a difference between the deleted and the proposed change. 0.4 vs 0.7 alpha. Is this a token regarding the White buttons?

**Pull request merged**